File: PHP/Program/GalaxySeries.php
<?php
require_once "Handphone.php";

class GalaxySeries extends Handphone {
    private $imei;
    private $model;
    private $category;

    // Constructor
    public function __construct($code, $name, $price, $androidVer, $uiVer, $processor, $imei, $model, $category, $photo) {
        parent::__construct($code, $name, $price, $androidVer, $uiVer, $processor, $photo);  // Menambahkan foto
        $this->imei = $imei;
        $this->model = $model;
        $this->category = $category;
    }

    // Setters and Getters
    public function setImei($imei) {
        $this->imei = $imei;
    }

    public function getImei() {
        return $this->imei;
    }

    public function setModel($model) {
        $this->model = $model;
    }

    public function getModel() {
        return $this->model;
    }

    public function setCategory($category) {
        $this->category = $category;
    }

    public function getCategory() {
        return $this->category;
    }
}
?>

File: PY/Program/GalaxySeries.py
from Handphone import Handphone

class GalaxySeries(Handphone):
    def __init__(self, code="", name="", price=0, android_ver="", ui_ver="", processor="", imei="", model="", category=""):
        super().__init__(code, name, price, android_ver, ui_ver, processor) # memanggil konstruktor dari kelas induk
        # Private attributes
        self.__imei = imei
        self.__model = model
        self.__category = category

    # getter dan setter untuk imei
    def get_imei(self):
        return self.__imei

    def set_imei(self, imei):
        self.__imei = imei

    # getter dan setter untuk model
    def get_model(self):
        return self.__model

    def set_model(self, model):
        self.__model = model

    # getter dan setter untuk category
    def get_category(self):
        return self.__category

    def set_category(self, category):
        self.__category = category
